docs/tutorials/lander.md

```diff
@@ -0,0 +1,340 @@
+    . . . . . . . 1 1 1
+    . . . 1 . . . 1 1 1
+    . . 1 1 1 . . 1 1 .
+    . . 1 1 1 . . . 1 .
+    . . 1 1 1 1 . . . .
+    . 1 1 1 1 1 1 . . .
+    7 1 1 1 1 1 1 1 1 4
+`)
+```
+
+## Step 3 @fullscreen
+
+Go into ``||sprites:Sprites||`` and grab a ``||sprites:set sprite of kind Player||`` and drag it into ``||loops:on start||``. Click the drop down for mySprite and rename the sprite to Lander. In the ``||sprites:sprite of kind Player||`` block, click on the grey box to open the **image editor**. Draw a spaceship.
+
+```blocks
+Lander = sprites.create(img`
+    . . . . . . . . . . . . . . . .
+    . . . . . . . . . . . . . . . .
+    . . . . . . . . . . . . . . . .
+    . . . . . . . . . . . . . . . .
+    . . . . . . d d d . . . . . . .
+    . . . . . d d d d d . . . . . .
+    . . . . d d 1 1 1 d d . . . . .
+    . . . . d d 1 1 1 d d . . . . .
+    . . . . d d d d d d d . . . . .
+    . . . . . d d d d d . . . . . .
+    . . . . . f d d d f . . . . . .
+    . . . . f f . . . f f . . . . .
+    . . . 2 . . . . . . . 2 . . . .
+    . . . . . . . . . . . . . . . .
+    . . . . . . . . . . . . . . . .
+    . . . . . . . . . . . . . . . .
+`, SpriteKind.Player)
+```
+
+## Step 4 @fullscreen
+
+Go into ``||sprites:Sprites||`` and grab a ``||sprites:set mySprite position to||`` and drag it under ``||sprites:set Lander to sprite of kind Player||``. Change ``||varibales:mySprite||`` to ``||variables:Lander||`` and then go into the position selector (click the space in x and choose the plus arrow) and place it above the green.  
+
+```blocks
+let Lander: Sprite = null
+Lander.setPosition(6, 102)
+```
+
+## Step 5 @fullscreen
+
+In ``||sprites:Sprites||`` grab a ``||sprites:set sprite stay in screen||`` and drag it under the position block. Toggle the switch to on and set the variable to Lander. Next grab a ``||sprites:set sprite x||`` and drag it under the stay in screen block. Set the drop down to **ay**, and the value to be **30**.
+
+```blocks
+let Lander: Sprite = null
+Lander.ay = 30
+Lander.setFlag(SpriteFlag.StayInScreen, true)
+```
+
+## Step 6 @fullscreen
+
+In ``||variables:Variables||`` grab a ``||variables:set to||`` block and drag it into on start. Change the variable name to angle and then drag a text box from ``||text:Text||`` and drag it into the value. Set the value to straight.
+
+```blocks
+let angle = ""
+angle = "straight"
+```
+
+## Step 7 @fullscreen
+
+Next drag an ``||controller:on A button pressed||`` from ``||controller:Controller||`` and drag it into your workspace. From ``||logic:Logic||`` grab an ``||logic:if then else||`` statement and drag it into ``||controller:on A button pressed||``. Then click the plus arrow. Grab a ``||logic:0=0||`` block from from ``||logic:Logic||`` and place two of them in the if else if sections. For both equal statements set the first value to ``||variables:angle||`` and for the first set the second field to ``||text:" "||`` of straight and the second to  ``||text:" "||`` of left. 
+
```

Comment:
With this step, I didn't want to put the snippet in until the next slide of the tutorial because I felt like having an incomplete block show up could be confusing. The if statement will be completely empty on the inside, should I still implement?

Reply:
I'm not sure this is any less complete than any other steps in tutorials; here I'm counting about 11 distinct actions needed to complete the step (that is, dragging a block, or modifying a block by clicking a button / filling in a text box), which feels like a bunch to do at once without some example to compare against when it's complete